packages/astro/src/types/public/config.ts

```diff
@@ -1379,6 +1380,54 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 		experimentalDefaultStyles?: boolean;
 	};
 
+	/**
+	 * @docs
+	 * @kind heading
+	 * @name SVG Options
+	 */
+	svg?: {
+		/**
+		 * @docs
+		 * @name svg.optimize
+		 * @type {boolean}
+		 * @default `true`
+		 * @description
+		 * Whether to enable SVG optimization using SVGO during build time.
+		 *
+		 * When enabled, all imported SVG files will be optimized for smaller file sizes
```

Comment:
> all imported SVG files

Will this also apply to usage with Astro’s assets pipeline? For example usage like:

```astro
---
import { Image } from 'astro:assets';
import mySVG from './example.svg';
---

<Image src={mySVG} alt="Example SVG" />
```